sdk/src/client/stronghold/common.rs

```diff
@@ -29,7 +31,7 @@ pub(super) const PRIVATE_DATA_CLIENT_PATH: &[u8] = b"iota_seed";
 pub(super) const USERDATA_STORE_KEY_RECORD_PATH: &[u8] = b"userdata-store-key";
 
 /// Hash a password, deriving a key, for accessing Stronghold.
-pub(super) fn key_provider_from_password(password: &str) -> KeyProvider {
+pub(super) fn key_provider_from_password(password: Password) -> KeyProvider {
```

Comment:
Into?

Reply:
Yeah, that's the reason. Another one is that the function name also suggests that we already have a `Password`.

Reply:
Fair enough